mta-sts.md

```diff
@@ -561,16 +610,15 @@ MTA-STS policy indicator TXT RR:
 _mta-sts.example.com.  IN TXT "v=STSv1; id=20160831085700Z;"
 ~~~~~~~~~
 
-MTA-STS Policy JSON served as the response body at
-https://mta-sts.example.com/.well-known/mta-sts.json:
+MTA-STS Policy file served as the response body at
+https://mta-sts.example.com/.well-known/mta-sts.policy:
```

Comment:
if it's going to be text/plain, can it be a .txt file so default config works most places (eg nginx and S3 serves a file called mta-sts.policy as application/octet-stream by default)?